### ISSUE TITLE
astool: no ontology for http://www.w3.org/TR/activitystreams-vocabulary

### ISSUE THREAD

**Issue by Zauberstuhl** [go-fed/activity#117]
**Please Describe The Problematic Behavior**

astool panics if executed with the example spec file.

```
> ./astool/astool -spec astool/activitystreams.jsonld -spec astool/example_custom_spec.jsonld ./streams
Auto-detected path: github.com/go-fed/activity
Reading input specifications...
Parsing 2 vocabularies...
panic: no ontology for https://www.w3.org/TR/activitystreams-vocabulary
```

**Please Describe The Expected Behavior**

no panics ;)

---

workaround: change `as` to `https://www.w3.org/ns/activitystreams`

**Comment by cjslep** [comment]
So generating with multiple files works with the activity streams and security ontologies:

`/Users/cjslep/go/bin/astool -spec ../astool/activitystreams.jsonld -spec ../astool/security-v1.jsonld .`

Guess I just need to look into the example file specifically.

**Comment by cjslep** [comment]
Fixed in cd97d44b995edbad434085b7b4a87880f8648531
